Blob QoL Update and Rebalance

Arkatos1 opened the pull request:
**What does this PR do:**
Includes large amount of Quality of Life changes to the Blob antagonist, varying from Blob UI improvements, aesthetics, rebalance, minor refactors to new features. Lets get through them, one by one:

1) Blob Mobs now use complementary colors of the Blob Overmind reagent - Previously, it was sometimes too hard to spot blob spores and blobbernauts if they were standing on blob tiles, because they were precisely same color as the rest of the blob tiles. This changes their color the to the complementary color of their reagent, so they will be clearly visible while maintaining sense of unity with the blob structures. Example image is below.

2) Storage Blob and Split consciousness abilities now have their UI buttons, along with a custom made icons. Example image is below.

3) Added Node Requirement verb to the Blob. Some Blob structures require to be in a close proximity to the core/node in order to work efficiently, and many new players have problem with that. This verb, which is activated by default, enforces that resource and factory blob structures are built in close proximity of the core/node, otherwise it will not allow them to build them with a message explaining why. Veteran players can disable the Node Requirement in the Blob tab if they wish to. Ported from /tg/station13.

4) Remove Blob verb now has a new feature - partial refund for special Blob structures. When Blob Overmind removes special blob structure (such as a factory or node), it will receive 30% of the structure cost back as a refund. This will give Blobs interesting options of tactical retreats from undefendeable areas, and also allows them to atleast partially remedy mistakes with bad special tile placements, for example. Ported from /tg/station13.

5) Dark Matter has been removed from a list of possible Blob reagents. Players have been turning chemical reroll into casino, waiting to hit that sweet jackpot, the Dark Matter, so they can fastmos their enemies to death and win. We currently have 8 different Blob reagents, so removing one is not much of a loss. We do not lose the entire mechanic either - Sorium uses the very same vortex ability, although it is milder and much more reasonable.

6) Minor refactor and improved descriptions Also added Blob Overminds to the Check Antagonists admin verb as requested.

Tested locally without any issue.

**Images of sprite/map changes (IF APPLICABLE):**
New UI, along with newly colored Blob Mobs:
![BetterBlobUI](https://user-images.githubusercontent.com/43862960/55269709-d1351600-5296-11e9-912f-6fe856e8b119.png)

**Changelog:**
:cl: Arkatos
add: New Blob UI buttons for Storage Blob and Split consciousness abilities
imageadd: New custom icons for Storage Blob and Split consciousness abilities
add: Added Node Requirement verb to the Blob. This verb, which is activated by default, enforces that resource and factory blob structures are built in a close proximity of the core/node in order to properly function. Players can disable it anytime in the Blob tab.
add: Remove Blob verb now has a new feature - partial refund for special Blob structures. When Blob Overmind removes special blob structure (such as a factory or node), it will receive 30% of the structure cost back as a refund. 
tweak: Blob Overminds now show in Check Antagonists admin verb 
tweak: Blob Mobs now use complementary colors of the Blob Overmind reagent as their own color
del: Removed Dark Matter from a list of possible Blob reagents
/:cl:

Mitchs98 commented:
Thank god this PR removes Dark Matter. It's by far the single most annoying and unbalanced thing about blob. Even worse considering people can just constantly roll for it until they get it. Dark Matter is most often a no effort win for blob.

Everything else looks quite nice as well :+1:

Kyep commented:
I have a request to make. Would you be able to add blob overminds to Check Antagonist?
I support this PR either way, I've just always wanted to see that particular change.

Fox-McCloud commented on code/modules/reagents/chemistry/reagents/blob.dm:
```diff
@@ -118,7 +118,7 @@
 		M.apply_damage(0.6*volume, BRUTE)
 
-
+*/
```
if you're going to remove something, don't comment it out.

Arkatos1 replied:
Fixed.

EvadableMoxie commented:
It's probably not fair to label a PR that removes darkmatter, and therefore has major balance implications as a 'QoL.' I hate dark matter so I'm in support of the PR.  I just think it should perhaps be more transparent that this is both QoL updates and a balance change.

KasparoVy commented:
FUUUUCK dark matter.

Arkatos1 commented:
I have renamed PR title to be more transparent, did some code cleanup and added Blob Overminds to the Check Antagonists verb.

@Kyep Done. Works both on Blob rounds and Blob midround.
